relay-profiling/src/lib.rs

```diff
@@ -75,10 +80,55 @@ struct MinimalProfile {
     #[serde(alias = "profile_id", alias = "chunk_id")]
     event_id: ProfileId,
     platform: String,
+    release: Option<String>,
```

Comment:
Will we filter on more fields in the future? Users might be surprised when they configure a release filter, see that it works, and then try the same for ip_addr and it fails.

Reply:
For `ip_addr`, I see that it's pulled from the user for other event types. Meaning for a backend event, it'll use the frontend user IP. Is this a strict requirement? Because for continuous profiles, we can only send the up of the server in some cases.

Reply:
So far the filter has only been applied to the end-user IP like you said, so in this case I would leave it empty (`None`) for continuous profiles.